pkg/services/business_case.go

```diff
@@ -210,20 +214,18 @@ func NewUpdateBusinessCase(
 				Operation: apperrors.QuerySave,
 			}
 		}
+
 		// Right now, add some validations for sending an email here
 		// Similar to system intake,
 		// these should be covered by validations above
 		if businessCase.Status == models.BusinessCaseStatusSUBMITTED &&
 			existingBusinessCase.Status == models.BusinessCaseStatusDRAFT {
-			if !businessCase.Requester.Valid {
-				validationError := apperrors.NewValidationError(
-					errors.New("failed to validate for email"),
-					businessCase,
-					businessCase.ID.String(),
-				)
-				validationError.WithValidation("Requester", "is required")
-				logger.Error("Failed to validate", zap.Error(&validationError))
-				return businessCase, &validationError
+			// Set submitted at time before validations as it is one of the fields that is validated
+			businessCase.SubmittedAt = businessCase.UpdatedAt
+			err := appvalidation.BusinessCaseForSubmit(businessCase, existingBusinessCase)
```

Comment:
I think we're going to need to do two blocks:
1. do submit validations, set submit time and save the business case (reusing above)
2. this one, where we just send the email

Same as with the system intake, the email should only happen if everything else succeeds.